docs/designs/2848-metapackage-proposal.md

```diff
@@ -0,0 +1,169 @@
+# Metapackage-based Installation Experience
+
+* Proposal: https://github.com/vmware-tanzu/community-edition/issues/2848
+
+## Summary
+
+In order to simplify the experience of installing multiple packages, we wanted to leverage the ability of creating a "metapackage" using Carvel tooling. 
+
+This metapackage is essentially a package of packages wherein it references a Bill of Materials that includes existing packages in the repository. Being able to install, update & remove these packages in concert with each other simplifies the complexity significantly. 
+
+This is a method in practice in other projects leveraging Carvel tooling. 
+
+## Motivation
+
+The motivation for introducing this is to reduce the complexity that comes up with installing multiple carvel packages at a time. This will allow newcomers to the project rapidly establish a working technology stack.
+
+* Describe why the change is important and the benefits to users.  
+
+### Goals
+
+* Demonstrate the ability that multiple subordinate packages can be installed using a single metapackage. 
+* Be able to provide a single values.yaml file that can be used for all the subordinate package values.yaml.
+
+### Non-Goals/Future Work
+
+* Define each profile (and the packages it will have) that will use this metapackage technology. 
+
```

Comment:
Can we call out a non-goal here as _not_ being able to coordinate packages when deployed?

For example, today, _i don't believe_ it's possible via Carvel for packages to introspect one another. So if I installed a prometheus, grafana observability profile, they wouldn't be able to introspect eachother intelligently to automatically deploy dashboards into grafana based on what's on the cluster.

Reply:
ref the idea of dependency management issue that could trend towards something like this being possible.
https://github.com/vmware-tanzu/carvel-kapp-controller/issues/371